index.js

```diff
@@ -0,0 +1,51 @@
+//const Books = [];
+const listLink = document.querySelector('#list-link');
+const addNewLink = document.querySelector('#add-new-link');
+const aboutLink = document.querySelector('#about-link');
+const listSection = document.querySelector('#list-section');
+const addNewSection = document.querySelector('#add-new-section');
+const aboutSection = document.querySelector('#about-section');
+const bookList = document.querySelector('#book-list');
+const addBookForm = document.querySelector('#add-book-form');
+import { handleAddBookFormSubmit } from './modules/bookForm.js';
+import { handleBookListClick, showNoBooksMessage } from './modules/bookList.js';
+import { updateDateTime } from './modules/dateandtime.js';
+
+updateDateTime();
+setInterval(updateDateTime, 1000);
+
+// Event listener for adding a book
+addBookForm.addEventListener('submit', (e) => {
+    handleAddBookFormSubmit(e, bookList, showNoBooksMessage);
+});
+
+// Event delegation for the "Remove" button
+bookList.addEventListener('click', (e) => {
+    handleBookListClick(e, bookList, showNoBooksMessage);
+});
+
+
+// Event listeners for navbar links
+listLink.addEventListener('click', () => {
+    displaySection(listSection);
+});
+
+addNewLink.addEventListener('click', () => {
+    displaySection(addNewSection);
+});
+
+aboutLink.addEventListener('click', () => {
+    displaySection(aboutSection);
+});
+
+// Function to display a section and hide the others
+const displaySection = (section) => {
+    const sections = [listSection, addNewSection, aboutSection];
+    sections.forEach((s) => {
+        if (s === section) {
+            s.style.display = 'block';
+        } else {
+            s.style.display = 'none';
+        }
+    });
+};
```

Comment:
- You have done great so far. :+1: :+1: Using ES6 syntax and luxon :+1: But there are some changes you need to make before moving to the next Milestone.
- [ ] Looking at the video below, you can see that when the website loads, the whole pages `book list, addbook form and about` are visible in the UI which is not meant to be, I'll suggest you fix this as its not good for the best user experience. :smile: :wave: 
- - Video:

https://github.com/Hsuwaddyhtat22/Awesome-Books/assets/107632780/86d6f470-53c9-4a0c-9014-c83afb79402a